test/controllers/api/v1/tickets_controller_test.rb

```diff
@@ -38,4 +37,17 @@ class Api::V1::TicketsControllerTest < ActionController::TestCase
     assert_response :success
   end
 
+  test 'should create ticket' do
+    sign_in users(:bob)
+    assert_difference 'Ticket.count', 1 do
+      post :create, auth_token: users(:bob).authentication_token, ticket: {
+        content: 'I need help',
+        from: 'bob@xxxx.com',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.